app/Models/Genre.php

```diff
@@ -8,4 +8,20 @@
 class Genre extends Model
 {
     use HasFactory;
+
+    public const GENRE_ACTION = 'Action';
+    public const GENRE_ANIME = 'Anime';
+    public const GENRE_ADVENTURE = 'Adventure';
+    public const GENRE_COMEDY = 'Comedy';
+    public const GENRE_CRIME = 'Crime';
+    public const GENRE_DRAMA = 'Drama';
+    public const GENRE_FANTASY = 'Fantasy';
+    public const GENRE_HORROR = 'Horror';
+    public const GENRE_MYSTERY = 'Mystery';
+    public const GENRE_ROMANCE = 'Romance';
+    public const GENRE_THRILLER = 'Thriller';
+
+    public function movies() {
+        return $this->belongsToMany('App\Movie');
```

Comment:
Same as the Celeb Model comment :D

`return $this->belongsToMany(Movie::class);`

Reply:
You can use the genre_movies pivot table as a second parameter too, 

```
return $this->belongsToMany(Movie::class, 'genre_movie', 'genre_id', 'movie_id');
```

That applies for all relations that have a pivot ;D

Reply:
Done :D